src/index.css

```
@@ -90,6 +90,81 @@ body {
   text-align: center;
 }
 
+@keyframes spin {
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+/* Auth screen animations */
+@keyframes bananaFall {
```

Stylelint (17.6.0)

[error] Expected keyframe name "bananaFall" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)